acm/odh-core/acm-observability/grafana-dashboards/edge-inference-health.json

```diff
@@ -98,13 +98,13 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(rate(rest_server_requests_total{cluster=~\"$cluster\"}[$__rate_interval]) * 60) by(namespace, path, method)",
+          "expr": "node_namespace_pod_container:container_cpu_usage_seconds_total:sum{cluster=\"near-edge-aws-ocp412\",namespace=~\"bike-rental-app|tensorflow-housing-app\"}",
```

Comment:
I guess the cluster should be selectable from a dropdown, rather than hardcoded? If someone deploys this themselves, their cluster names will likely be different. Also ours are likely to change also, given that they've got cloud and OCP version info in them. Maybe the namespace should be selectable also. WDYT?

Reply:
Yep I think this is good, I never done that before with grafana but I will try it out 👍

Reply:
Updated it to have a dropdown for cluster and namespace